handsontable/src/plugins/contextMenu/contextMenu.scss

```
@@ -80,14 +80,19 @@
 }
 
 .htContextMenu table tbody tr td .htItemWrapper {
-  margin-left: 10px;
-  margin-right: 6px;
+  margin-inline-start: 10px;
+  margin-inline-end: 6px;
 }
 
 .htContextMenu table tbody tr td div span.selected {
   margin-top: -2px;
   position: absolute;
-  left: 4px;
+  inset-inline-start: 4px;
+}
+
+[dir=rtl].handsontable .htContextMenu table tbody tr td div span.selected {
+  inset-inline-start: unset;
+  inset-inline-end: 4px;
```

```suggestion
  inset-inline-start: 4px;
  inset-inline-start: 0px;
}
```
The logical CSS prop does the job here. I think that the `[dir=rtl]` selector is unnecessary.